### PR TITLE
1141: Add code include, don't show generation action if readme exists

### PR DESCRIPTION
**Pull request by makzef**
**Description** (*)

Implemented next solutions:

- Added check, if the [README.md](http://readme.md/) file is already generated do not show action.
- Added a new code include, to be able generate project-specific [README.md](http://readme.md/) files

**Fixed Issues (if relevant)**

1. Fixes magento/magento2-phpstorm-plugin#1141

**Contribution checklist** (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with integration/functional tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: resources/fileTemplates/includes/MD File Description.md.ft [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!---
+    You can describe your module here.
+    We recommend that you add the following information:
+     - implementation details: why and how to use the module, preferably with some example scenarios
+     - any dependencies (usually other modules but could be any important dependencies, libraries, etc)
+     - extension points, APIs, plug-ins, etc
+     - any introduced events
+-->
```
Comment (bohdan-harniuk) [inline resources/fileTemplates/includes/MD File Description.md.ft:8]: Please, add an empty line at the end of the file

Comment (bohdan-harniuk) [reply resources/fileTemplates/includes/MD File Description.md.ft:8]: + add appropriate description to the resources/fileTemplates/includes/MD File Description.md.html

File: src/com/magento/idea/magento2plugin/actions/context/md/NewReadmeMdAction.java [hunk 2]
```diff
@@ -57,4 +59,17 @@ protected AttributesDefaults getProperties(
 
         return defaults;
     }
+
+
```
Comment (bohdan-harniuk) [inline src/com/magento/idea/magento2plugin/actions/context/md/NewReadmeMdAction.java:63]: Remove an extra empty line

**Review by bohdan-harniuk — CHANGES_REQUESTED** [review]
Hello, @makzef !
Please check my suggestions.
Regards,

File: resources/fileTemplates/includes/MD File Description.md.ft (hunk 1 above)
Comment (makzef) [reply resources/fileTemplates/includes/MD File Description.md.ft]: Done.

**Review by bohdan-harniuk — APPROVED** [review]
Great work!

**Pull request merged**